04_HTML/Praktikum/CreateAccount.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Create Account</title>
+</head>
+
+<body>
+    <form>
+        <h1>Create Account</h1>
+        <h2>Detail Form</h2>
+
+        <p>First name :</p>
```

Comment:
seharusnya ini menggunakan element label dg for attribute

Reply:
mas ini berarti bagusnya di merge ke main dulu baru revisi. apa langsung revisi aja mas, nanti pull req lagi